pkg/controllers/plugin/pluginpreset_controller_test.go

```diff
@@ -242,6 +242,260 @@ var _ = Describe("PluginPreset Controller Lifecycle", Ordered, func() {
 	})
 })
 
+var _ = Describe("Plugin Preset skip changes", Ordered, func() {
```

Comment:
This feels like a table test...
Have a look [here](https://github.com/cloudoperators/greenhouse/blob/main/pkg/controllers/propagation_reconciler_test.go#L141-L244)

Reply:
And maybe we can make use of and extend the [test utilities / resources](https://github.com/cloudoperators/greenhouse/blob/main/pkg/test/resources.go)

Find [some examples in these tests](https://github.com/cloudoperators/greenhouse/blob/main/pkg/controllers/teamrbac/teamrolebinding_controller_test.go)

Reply:
Done